src/Presentation/Nop.Web/Themes/DefaultClean/Views/Shared/Head.cshtml

```diff
@@ -11,7 +11,9 @@
     //add main CSS file
     NopHtml.AppendCssFileParts($"~/Themes/{themeName}/Content/css/styles{(supportRtl ? ".rtl" : "")}.css");
 
+    //add swiper css file
+    NopHtml.AppendCssFileParts("~/lib_npm/swiper/swiper-bundle.min.css");

```

Comment:
Inject `CatalogSettings` and check whether the feature is enabled and we should add CSS for it.